development/development_norms.md

```diff
@@ -0,0 +1,12 @@
+# Contributor Development Guidelines
+
+The Drud project has a number of development norms which we try to state explicitly when we can:
+
+* **Use PRs to contribute**: Even when contributors have the privilege to directly commit to a repo, they typically contribute via a PR instead, which allows everybody to review and be aware of changes, even when they're very small. ([Github docs](https://help.github.com/categories/collaborating-with-issues-and-pull-requests/))
+* **Fork-and-branch PRs**: We typically use the fork-and-branch PR technique, where every contributor has their own fork of the main repo, and branches on the fork are used for PRs. This allows the contributor full control of their own repo. ([Github docs](https://help.github.com/articles/working-with-forks/))
+* **PR Naming** (and commits): Please name PRs so that people can (mostly) understand what they are about by just reading the title. Remember that people don't have time to click through every link, and you want them to know what's going on with a one-liner where possible. So "Restart ddev router after rebuilding db container, fixes #998" instead of "Restart router", for example. When possible, it's best for commits to have "real" summary/message as well, not "fix bug". We all know there are experimental times when this becomes difficult.
+* **PR Review is required**: With few exceptions, PRs require review before they are pulled. Significant PRs require review by at least two team members. These guidelines are generally followed even if the repo is not fully set up to enforce them.
+* **Review means manual testing AND code review**: It's too easy to just say "LGTM". Nearly every PR needs to be compiled and tested manually in the review process. (The PR template has a section for describing a manual testing procedure, currently titled "The Test". The PR creator should provide detailed instructions on how to test it.)
+* **PR contributor pulls own PR if they have privs to do so**: We've found that it works best for a PR initiator to pull their own PR after reviews are done and tests pass. The initiator typically knows any outstanding issues that need to be taken care of, and knows the best time for the PR to go in.
```

Comment:
Would it be worth outlining what happens when a contributor does not have the privileges to merge their own code?